configuration/appframework/scripts/patientHasEncounterOfTypeDuringProgramEnrollment.js

```diff
@@ -0,0 +1,8 @@
+function patientHasEncounterOfTypeDuringProgramEnrollment(encounterTypeUuid, programUuid) {
+
+  var isPatientEnrolled = isPatientEnrolledInProgramOfUuid(programUuid);
+  if ( !isPatientEnrolled ) {
+    return false;
+  }
+  return isPatientEnrolledInProgramOfUuid(programUuid) && encounterOfTypeAfterDate(encounters, encounterTypeUuid, getDateEnrolledByProgramUuid(activePrograms, programUuid));
```

Comment:
You don't need to evaluate `isPatientEnrolledInProgamOfUuid` again here, you already did it above.  Can't we just have this final statement?